app/models/tag.rb

```diff
@@ -153,10 +153,9 @@ def categorization
 
   # @return [ActiveRecord::Relation] Scope for tags controlled by ContainerLabelTagMapping.
   def self.controlled_by_mapping
-    queries = ContainerLabelTagMapping::TAG_PREFIXES.collect do |prefix|
-      where(arel_table[:name].matches("#{sanitize_sql_like(prefix)}%", nil, true)) # case sensitive LIKE
-    end
-    queries.inject(:or).read_only.is_entry
+    is_entry.where(:id => ContainerLabelTagMapping.includes(:tag => :classification).map do |m|
+      m.tag.classification.entries.map(&:tag_id)
```

Comment:
This could be a performance issue, need to investigate

Reply:
There might be a way to do this with scopes directly instead of an id query (though I'm not sure if `.entries` messes that up)